### DISCUSSION TITLE
Лимит на отображение треков.

### DISCUSSION THREAD

**Discussion by wugenz** [Chimildic/YaMuTools#106] [Помощь с ошибкой]
ну если не тянуть, то я пиздец какой меломан и как только моя библиотека (с расширением включая заблокированные треки) приближалась к рекордным для меня, да думаю вообще для кого-либо 10 тысячам в библиотеке избранных счетчик остановился на 9922 треках. что не пытался делать дефолтные перезагрузки страниц и пк ему ваще похуй. ну как-то так
![image](https://github.com/user-attachments/assets/520a4b5d-5295-4433-bbf4-19b8ffeb8ef4)

upd. так будет даже лучше
![image](https://github.com/user-attachments/assets/40c2bd0a-2cc2-4366-8742-8b08fe1cbe62)

upd2. я просто хз в какой раздел писать с этой темой, по этому написал сюда.

**Comment by Chimildic** [comment]
Из-за недоступных треков количество может быть не совсем верным. Разные способы подсчета дают разные числа. Скорее всего реальное количество равно 10 тысячам. Счетчик встал на месте потому что 10к это лимит для плейлистов яндекса. Теперь при добавлении новых лайков, он снимает старые лайки.

Нажмите _три точки_ > _добавить в плейлист_ > _название для нового._ Так создадите копию для хранения лайков.

**Reply by wugenz** [reply]
эх....
